CHANGELOG.md

```diff
@@ -1,3 +1,31 @@
+# dbt_shopify_holistic_reporting v0.11.0
+
+[PR #35](https://github.com/fivetran/dbt_shopify_holistic_reporting/pull/35) includes the following updates:
+
+## Breaking Changes (--full-refresh required after upgrading)
+- Increases the upstream `dbt_shopify` dependency from `[">=0.21.0", "<0.22.0"]` → `[">=1.4.0", "<1.5.0"]`. Refer to the below Shopify releases to understand the full scope of changes:
+  - [dbt_shopify v0.22.0](https://github.com/fivetran/dbt_shopify/releases/tag/v0.22.0)
+  - [dbt_shopify v1.0.0](https://github.com/fivetran/dbt_shopify/releases/tag/v1.0.0)
+  - [dbt_shopify v1.1.0](https://github.com/fivetran/dbt_shopify/releases/tag/v1.1.0)
+  - [dbt_shopify v1.2.0](https://github.com/fivetran/dbt_shopify/releases/tag/v1.2.0)
+  - [dbt_shopify v1.3.0](https://github.com/fivetran/dbt_shopify/releases/tag/v1.3.0)
+  - [dbt_shopify v1.3.1](https://github.com/fivetran/dbt_shopify/releases/tag/v1.3.1)
+  - [dbt_shopify v1.4.0](https://github.com/fivetran/dbt_shopify/releases/tag/v1.4.0) # TO BE RELEASED -- WAIT FOR THIS BEFORE MERGING
```

Comment:
Reminder to hold off on merging this PR until the upstream Shopify update is rolled out. Also reminder to remove this note in the CHANGELOG.